### ISSUE TITLE
[no-unsafe-member-access] location of warning is swamping other warnings

### ISSUE THREAD

**Issue by LakMoore** [typescript-eslint/typescript-eslint#2164]
```JS
{
  "rules": {
      '@typescript-eslint/no-unsafe-member-access': 'error',
      '@typescript-eslint/no-unsafe-call': 'error',
  }
}
```

```TS
  public getObs$(): Observable<number> {
    return new Observable<number>();
  }

  public getPopularDepartments(): void {
    this.getObs$.pipe().subscribe((res) => {
      console.log(res);
    });
  }
```

**Expected Result**
```TS
  public getPopularDepartments(): void {
    this.getObs$.pipe().subscribe((res) => {
                 ~~~~   ~~~~~~~~~
      console.log(res);
    });
  }
```
Two separate warnings

**Actual Result**
```TS
  public getPopularDepartments(): void {
    this.getObs$.pipe().subscribe((res) => {
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      console.log(res);
    });
  }
```
By appearances, a single error, with the true error hidden within

**Additional Info**
The plugin could probably do with a 'loc' property to be returned to define exactly where to show the warning/error.  In some paths, the code finds the name of the offending method (subscribe) in this example.  It might be a case of finding that string within the source and returning that as the loc property.

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `3.0.2` |
| `@typescript-eslint/parser`        | `3.0.2` |
| `TypeScript`                       | `3.8.3` |
| `ESLint`                           | `7.1.0` |
| `node`                             | `12.16.3` |
| `npm`                              | `6.14.4` |

**Comment by LakMoore** [comment]
![shop_service_ts_—_Client](https://user-images.githubusercontent.com/6624290/83652387-7988ed80-a5b2-11ea-8a64-3e2f0ea52ece.jpg)

**Comment by bradzacher** [comment]
happy to accept a PR if you'd like to focus the report location!
